bindings/matrix-sdk-ffi/src/authentication_service.rs

```diff
@@ -556,6 +558,7 @@ impl AuthenticationService {
             .map_err(|_| AuthenticationError::OidcCallbackUrlInvalid)?;
         let client_name = Some(Localized::new(configuration.client_name.to_owned(), []));
         let client_uri = Url::parse(&configuration.client_uri).ok().map(|u| Localized::new(u, []));
+        let logo_uri = Url::parse(&configuration.logo_uri).ok().map(|u| Localized::new(u, []));
```

Comment:
```suggestion
        let logo_uri = configuration
            .logo_uri
            .as_deref()
            .map(Url::parse)
            .transpose()
            .map_err(|_| AuthenticationError::OidcMetadataInvalid)?
            .map(|u| Localized::new(u, []));
```

I'm not sure about the error to return though.

Reply:
That error looks correct to me 👍